src/app/modules/gb-cohort-selection-module/constraint-components/gb-combination-constraint/gb-combination-constraint.component.html

```diff
@@ -5,7 +5,7 @@
     <p-dropdown [options]="dimensions" [(ngModel)]="selectedDimension"
                 [disabled]="dimensions.length === 1"
                 [dropdownIcon]="dimensions.length === 1 ? '' : 'pi pi-chevron-down'"
-                (onChange)="onSelectedDimensionChange()" [autoWidth]="false" [style]="{'min-width':'10em'} ">
+                (onChange)="onSelectedDimensionChange()" [style]="{'min-width':'10em'} ">
```

Comment:
was this in conflict with the version updates?

Reply:
indeed, the newer primeng does not provide this attribute.

Reply:
But let me check again, because only the patch version changed.

Reply:
Just checked and this change is indeed not required. Probably leftover from when I tried to upgrade primeng.